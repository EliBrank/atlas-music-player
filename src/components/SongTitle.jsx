const SongTitle = ({artistName, children}) => {
  return (
    <div>
      <h2 className="mb-2 text-2xl font-bold">{children}</h2>
      <p className="text-grey">{artistName}</p>
    </div>
  );
}

export default SongTitle;
