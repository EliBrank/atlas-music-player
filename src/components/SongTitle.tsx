type SongTitleTypes = {
  songName: string,
  artistName: string,
}

const SongTitle = ({songName, artistName}: SongTitleTypes) => {
  return (
    <div>
      <h2 aria-label="Now Playing" className="mb-2 text-2xl font-bold">{songName}</h2>
      <p className="mb-4 text-tertiary-light dark:text-tertiary-dark">{artistName}</p>
    </div>
  );
}

export default SongTitle;
