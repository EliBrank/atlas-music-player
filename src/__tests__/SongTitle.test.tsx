import { render } from "@testing-library/react";
import { expect, test } from "vitest";
import SongTitle from "@/components/SongTitle";

test("SongTitle renders correctly", () => {
  const { container } = render(
    <SongTitle
      songName="Testing Once Again"
      artistName="The Testers"
    />
  );
  expect(container).toMatchSnapshot();
});
