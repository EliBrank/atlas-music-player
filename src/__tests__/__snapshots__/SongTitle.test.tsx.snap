// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SongTitle renders correctly 1`] = `
<div>
  <div>
    <h2
      aria-label="Now Playing"
      class="mb-2 text-2xl font-bold"
    >
      Testing Once Again
    </h2>
    <p
      class="mb-4 text-tertiary-light dark:text-tertiary-dark"
    >
      The Testers
    </p>
  </div>
</div>
`;
